subprojects/perlin_noise.wrap

```diff
@@ -0,0 +1,7 @@
+[wrap-git]
+url = https://github.com/Matthieu-Sauer/PerlinNoise.git
+revision = master

```

Comment:
Please specify a specific version or commit here.
It would be bad if behaviour changed by itself.

Reply:
Change will be implemented :)